### ISSUE TITLE
Add Captcha in the sign-in flow

### ISSUE THREAD

**Issue by mikebudzynski** [Azure/api-management-developer-portal#952]

**Comment by ilspiert** [comment]
Good idea. But I would appreatiate if you could make that configurable even in the managed Portal. Thx.

**Comment by DarinMacRae** [comment]
Has anyone made any progress on this, either inside or outside of Microsoft? I've looked around for a custom widget or some post regarding it but so far haven't found anything.
